net/mosh/Portfile

```diff
@@ -4,8 +4,8 @@ PortSystem              1.0
 PortGroup               perl5 1.0
 
 name                    mosh
-version                 1.3.2
-revision                18
+version                 1.4.0
+revision                0
```

Comment:
Nit: revision 0 is the default, so you can just omit it now.

Reply:
I'm aware of that, but I think there was some discussion on the macports-dev list recently where we decided that keeping it in and set to 0 is not recommended over removing it.

Reply:
because the revision inevitably increases for so many ports, and it can be tricky to put it in the right spot, and we all hoped for some automation to manipulate the revision when revbumping often large groups of ports, we decided to leave it in, even when resetting to zero…

Reply:
Gotcha, I didn't see that thread go by. *shrug*